core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -975,7 +975,7 @@ public String getOSDescription() throws IOException, InterruptedException {
     private static final class SlaveVersion extends MasterToSlaveCallable<String,IOException> {
         public String call() throws IOException {
             try { return Launcher.VERSION; }
-            catch (Throwable ex) { return "< 1.335"; } // Older slave.jar won't have VERSION
+            catch (Throwable ex) { return "< 1.335"; } // Older agent.jar won't have VERSION
```

Comment:
isn't it still called slave.jar or was that renamed?

Reply:
Both names/URLs exist, this is the preferred one.

https://github.com/jenkinsci/jenkins/blob/62546443947fafecc94a5e3b47bb9129bed8e148/core/src/main/java/hudson/model/Slave.java#L732